react/components/Buttons/AddToCart.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Button from 'vtex.styleguide/Button'
+import { injectIntl, intlShape } from 'react-intl'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+
+class AddToCart extends Component {
+  static propTypes = {
+    total: PropTypes.number,
+    intl: intlShape.isRequired,
+  }
+
+  render() {
+    const intl = this.props.intl
+    const label = intl.formatMessage({ id: 'product-customizer.add-to-cart' })
+
+    return (
+      <div className={'actions--add-to-cart tc ph5 pb5'}>
```

Comment:
Remove useless curly brackets. :pick:

Reply:
Fixed in commit 2d2092c522f572a871eea9c57d10a73b4837ee5f